feat(changelog): environmental-footprint-added-environmental-footprint-c 2025-01-30 (#4329)
scaleway/docs-content: changelog/january2025/2025-01-30-environmental-footprint-added-environmental-footprint-c.mdx

---
title: Environmental footprint Calculator is now in Public Beta allowing you to track and understand the environmental impact of your cloud resources 🌱
status: added
author:

>     fullname: '#environmental-footprint-'

Reviewer: ```suggestion
    fullname: 'Join the #environmental-footprint channel on Slack.'
```

    url: 'https://slack.scaleway.com'
date: 2025-01-30
category: environmental-footprint
product: environmental-footprint
---

> Currently available on Elastic Metal offers, the calculator will then integrate all Scaleway products. Refere to the https://www.scaleway.com/en/docs/environmental-footprint/environmental-footprint/  for more information.

Reviewer: ```suggestion
The Environmental Footprint calculator is currently available for Elastic Metal offers. The calculator will progressively integrate all Scaleway products. Refer to the [Environmental Footprint documentation](https://www.scaleway.com/en/docs/environmental-footprint/)  for more information.
```

> The Environmental Footprint calculator is currently available for Elastic Metal offers. The calculator will progressively integrate all Scaleway products. Refer to the [Environmental Footprint documentation](https://www.scaleway.com/en/docs/environmental-footprint/)  for more information.

Reviewer 2: ```suggestion
The Environmental Footprint calculator is currently available for Elastic Metal offers. The calculator will progressively integrate all Scaleway products. Refer to the [Environmental Footprint documentation](/environmental-footprint/)  for more information.
```

> The Environmental Footprint calculator is currently available for Elastic Metal offers. The calculator will progressively integrate all Scaleway products. Refer to the [Environmental Footprint documentation](https://www.scaleway.com/en/docs/environmental-footprint/)  for more information.

Author: ```suggestion
The Environmental Footprint calculator is currently available for Elastic Metal offers. The calculator will progressively integrate all Scaleway products. For more information, refer to the [Environmental Footprint documentation](https://www.scaleway.com/en/docs/environmental-footprint/).
```
